virtual_namespace.py

```diff
@@ -65,9 +69,12 @@ def __init__(self, code, name):
     # Remove any windows carriage returns
     code = code.replace('\r\n','\n')
 
-    # Prepend an encoding string to protect against bugs in that code (ticket #982)
-    # Note that this will cause tracebacks to have an inaccurate line number.
-    code = "# coding: utf-8\n\n" + code 
+    # Prepend an encoding string to protect against bugs in that code (#982).
+    # Without further fixes, prepending the encoding string causes tracebacks 
+    # to have an inaccurate line number, see SeattleTestbed/repy_v2#95.
+    # We work around this in tracebackrepy.py
```

Comment:
We also seem to work around this in `safe_code.py` and others.  Not sure if it's worth mentioning those as well.